packages/react-api/src/hooks/FeaturesDroppedLoader.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  name: 'FeaturesDroppedLoader',
```

Comment:
👍🏻 I was curious about the approach... How does this work without interfering with other loaders? Is it some kind of wrapper or interceptor?

Reply:
It works by invoking `context.parse` after extracting the header. In deck.gl we have our own loaders which actually parse the binary data, which [are registered](https://github.com/visgl/deck.gl/blob/master/modules/carto/src/layers/carto-tile-layer.ts#L2-L3) as supporting our custom mimeType. Because `FeaturesDroppedLoader` specifies a mimeType of `application/vnd.carto-vector-tile` it will only be invoked when the returned data matches that. So it shouldn't interfere with the other loaders